oecophylla/taxonomy/taxonomy.rule

```diff
@@ -309,13 +301,5 @@ rule shogun:
 rule taxonomy:
     input:
         taxonomy_dir + "metaphlan2/joined_taxonomic_profile.tsv",
-        # taxonomy_dir + "kraken/combined_profile.tsv",
-        # taxonomy_dir + "shogun/combined_profile.tsv",
-
-        # the following two lines are for Kraken, Bracken and SHOGUN
-        # once the biom function is in place, they should be edited
-        expand(taxonomy_dir + "{sample}/kraken/{sample}.map.txt", sample=samples),
-        expand(taxonomy_dir + "{sample}/kraken/{sample}.report.txt", sample=samples),
-        expand(taxonomy_dir + "{sample}/bracken/{sample}.table.txt", sample=samples),
-        expand(taxonomy_dir + "{sample}/bracken/{sample}.report.txt", sample=samples),
-        expand(taxonomy_dir + "{sample}/shogun/{sample}.taxon_counts.tsv", sample=samples)
+        taxonomy_dir + "kraken/combined_profile.biom"
+        # taxonomy_dir + "shogun/combined_profile.tsv"
```

Comment:
Why commented out here?

Reply:
Because in this PR and so on I am trying to have Stefan's parser to combine tables. Currently it is connected to the Kraken rule, but not the SHOGUN rule. So I commented it out.